DopplerCustomDomain/CustomDomainProvider/CustomDomainProviderService.cs

```diff
@@ -0,0 +1,35 @@
+using Flurl.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Tavis.UriTemplates;
+
+
+namespace DopplerCustomDomain.CustomDomainProvider
+{
+    public class CustomDomainProviderService : ICustomDomainProviderService
+    {
+        private readonly ILogger<CustomDomainProviderService> _logger;
+
+        public CustomDomainProviderService(ILogger<CustomDomainProviderService> logger)
+        {
+            _logger = logger;
+        }
+
+        public Task<List<string>> CreateAllCustomDomains()
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public Task CreateCustomDomains(string domain)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public Task DeleteCustomDomains(string domain)
+        {
+            throw new System.NotImplementedException();
+        }
```

Comment:
These names should be in singular, right?

```suggestion
        public Task CreateCustomDomain(string domain)
        {
            throw new System.NotImplementedException();
        }

        public Task DeleteCustomDomain(string domain)
        {
            throw new System.NotImplementedException();
        }
```